resources/views/shop/components/comment.blade.php

```diff
@@ -0,0 +1,32 @@
+@forelse($reviews as $review)
+<div class="row mb-5">
+    <div class="col-md-2">
+        <div class="user-thumbnail">
+            <img class="user-photo" src="{{ asset('/img/users/default.png') }}"
+            alt="Avatar">
+        </div>
+    </div>
+
+    <div class="col-md-10">
+        <div class="card">
+            <div class="card-header comment-header">
+                <div>
+                    <strong>{{ $review->user->full_name }}</strong>
+                    <span class="text-muted">
+                        @lang('common.text.shop_page.single_product_page.commented_at')
+                        {{ $review->created_at->diffForHumans() }} :</span>
+                </div>
+                <div>
+                    @for ($i = 1; $i <= @config('setting.product.number_rating'); $i++)
```

Comment:
đoạn code này dùng làm gì đây em

Reply:
Đoạn này em dùng để in ra số sao vàng(hoặc đen) dựa trên lượt rate của user cho 1 product. User rate 1 product là 3 sao thì sẽ in ra 3 sao vàng còn lại là 2 sao đen và tổng số sao sẽ tùy ý mình config trong setting.